src/dev/i18n/extract_default_translations.js

```diff
@@ -38,15 +40,56 @@ function addMessageToMap(targetMap, key, value) {
   targetMap.set(key, value);
 }
 
-export async function extractDefaultTranslations(inputPath) {
+async function getPluginsPaths(inputPaths) {
+  const availablePaths = Object.values(config.paths);
+  const pathsForExtraction = [];
+
+  for (const inputPath of inputPaths) {
+    const normalizedPath = normalize(path.relative('.', inputPath));
+
+    if (normalizedPath) {
+      pathsForExtraction.push(
+        ...availablePaths.filter(
+          ePath => ePath.startsWith(`${normalizedPath}/`) || ePath === normalizedPath
+        )
+      );
+    } else {
+      pathsForExtraction.push(...availablePaths);
```

Comment:
__question:__ in what cases `normalizedPath` is falsy? Also could you please describe a little bit what you're doing here, I'm not sure I'm entirely following the idea.

Reply:
`path.relative('.', './') === ''`.
`normalizedPath === ''` means that the tool should parse messages from all known paths.

Reply:
But can this really happen? I see there is a `./` default if `path` isn't provided here: https://github.com/elastic/kibana/pull/21588/files/31a074daaac3c15cd90502c0e427540eef77905e#diff-46efc3ab1fc36e619f075d52521be553R28

Reply:
Yes, it can happen. You have provided a nice example :) If `path` is skipped, then the tool should extract messages from all paths in `config.paths`.